docs/user-guide/config-examples.md

```diff
@@ -0,0 +1,43 @@
+# User configurations
+
+Want some help with starting your configuration?
+Or maybe you're curious how others have done things?
+Take a look at these configuration examples below.
+
+<!-- TODO: add info about `kickstart.nixvim` -->
+
+**Note:**\
+Most of those configurations are using a [standalone build](../modules/standalone.html), however,
+all of the nixvim options are accessible no matter how you are using it (flake, NixOS/HM module, nix-darwin...).
+
+| Config | Comment |
+|-|-|
+| [ahwxorg/nixvim-config] | |
+| [alisonjenkins/neovim-nix-flake] | |
+| [GaetanLepage/nix-config] | Home-manager |
+| [gwg313/nvim-nix] | |
+| [hbjydev/hvim] | |
+| [nicolas-goudry/nixvim-config] | Heavily inspired by AstroNvim |
+| [pete3n/nixvim-flake] | |
+| [redyf/Neve] | meticulously crafted custom configuration for Nixvim |
+| [siph/nixvim-flake] | |
+| [Tanish2002/neovim-config] | |
+| [traxys/Nixfiles] | |
+
+[ahwxorg/nixvim-config]: https://github.com/ahwxorg/nixvim-config
+[alisonjenkins/neovim-nix-flake]: https://github.com/alisonjenkins/neovim-nix-flake
+[GaetanLepage/nix-config]: https://github.com/GaetanLepage/nix-config/tree/master/home/modules/tui/neovim
+[gwg313/nvim-nix]: https://github.com/gwg313/nvim-nix
+[hbjydev/hvim]: https://github.com/hbjydev/hvim
+[nicolas-goudry/nixvim-config]: https://github.com/nicolas-goudry/nixvim-config
+[pete3n/nixvim-flake]: https://github.com/pete3n/nixvim-flake
+[redyf/Neve]: https://github.com/redyf/Neve
+[siph/nixvim-flake]: https://github.com/siph/nixvim-flake
+[Tanish2002/neovim-config]: https://github.com/Tanish2002/neovim-config
+[traxys/Nixfiles]: https://github.com/traxys/Nixfiles/tree/master/neovim
```

Comment:
Traditionally, the "footnote" part of the link would go at the end of the document. But I don't think that's actually a requirement and I can see why you might prefer to keep it "local".

Personally, I'd have it at the EOF though.

Reply:
It seems to work fine like that.
Keeping it local makes more sense to me (as long as it works ofc).

Reply:
No strong opinion here. @traxys?

Reply:
No strong opinions from me either